### PR TITLE
What we do page

### PR DESCRIPTION
**Pull request by nicolemors**
Added in new what we do page. 
Moved icons out of the sandbox into top level of images within the services directory. 

screen shot 2016-06-28 at 1 59 21 pm

@tylersticka @aileenjeffries @mrgerardorodriguez @saralohr

### CODE REVIEW

**Comment by nicolemors** [comment]
<a href="https:&#x2F;&#x2F;trello.com&#x2F;c&#x2F;rkRDLjRz&#x2F;150-sketch-mockup-prototype-what-we-do"> Sketch&#x2F;Mockup&#x2F;Prototype: What We Do</a>

File: src/pages/pages/what_we_do.hbs [hunk 1]
```diff
@@ -0,0 +1,97 @@
+---
+title: What We Do
+layout: toolkit.blank
```
Comment (tylersticka) [inline src/pages/pages/what_we_do.hbs]: Nit: Add `notes` to summarize what this page's purpose is.

File: src/pages/pages/what_we_do.hbs [hunk 2]
```diff
@@ -0,0 +1,97 @@
...
+      </p>
+      <p>
+        Many companies know they need a responsive web design or progressive web app, but don't know where to start. Their teams face the pressure of deadlines but need the project done right the first time.
+      </p>
+      <p>
+        We work closely with internal teams to find the best design and technical approach. We set them up for success by incorporating education and training into every project.
+      </p>
+    </div>
+
+    <div class="Grid Grid--withGutter u-spaceItems1 u-containSpread u-pad1">
```
Comment (tylersticka) [inline src/pages/pages/what_we_do.hbs]: `u-pad1` and `Grid--withGutter` contradict each other. Consider separating these out into separate divs, perhaps one to contain and pad and the grid for layout.

**Comment by tylersticka** [comment]
One comment inline. Also, I would recommend removing the sandbox prototype file entirely if it duplicates what's in pages (it looks like it's been updated here but not removed).

**Pull request merged**